scripts/ci/check_compliance.py

```diff
@@ -927,6 +927,7 @@ def check_no_undef_outside_kconfig(self, kconf):
                               # Zephyr toolchain variant and therefore not
                               # visible to compliance.
         "BOARD_", # Used as regex in scripts/utils/board_v1_to_v2.py
+        "BOARD_REVISION", # Used in some Kconfig.board
```

Comment:
@nordicjm @tejlmand any idea on why this is required? see https://github.com/zephyrproject-rtos/zephyr/actions/runs/10919646498/job/30307751818

Reply:
Would assume the hierarchy of how the script runs, it probably just sources board files and not zephyr's Kconfig files

Reply:
Actually, this was not a fix, but https://github.com/zephyrproject-rtos/zephyr/pull/78631/commits/fe26d9e4752013681703ee50f688d8450ea87c6a (not sure if the best way, but seems to fix the problem).